src/component/button/button.css

```
@@ -0,0 +1,8 @@
+.button--small {
+  width: 100px;
+}
+
+.button--medium {
+  width: 100%;
+  max-width: 1280px;
+}
```

small - medium 으로 구성되어 있어 계층적으로 버튼의 크기가 점점 커지는 것을 예상했으나 실제 css는 그와 다른데요.

조금 더 예측 가능한 형태로 클래스명을 구성해보셔도 좋을거 같아요. 밑은 예시입니다
```suggestion
.button--small {
  width: 100px;
}

.button--medium {
  width: 120px;
}

.button--full {
  width: 100%;
  max-width: 1280px;
}
```